shared/src/main/java/uk/gov/di/authentication/shared/entity/CodeRequestType.java

```diff
@@ -64,12 +67,12 @@ public boolean equals(Object obj) {
                 return false;
             }
             CodeRequestTypeKey other = (CodeRequestTypeKey) obj;
-            return notificationType == other.notificationType && journeyType == other.journeyType;
+            return mfaMethodType == other.mfaMethodType && journeyType == other.journeyType;
         }
 
         @Override
         public int hashCode() {
-            return 31 * notificationType.hashCode() + journeyType.hashCode();
+            return 31 * mfaMethodType.hashCode() + journeyType.hashCode();
```

Comment:
Because we're changing the key in the CodeRequestType, will that make any requests made under the old key for the CodeRequestType redundant?

Reply:
Dom and Ayo have walked me through this and believe there will be no impact.